src/mapkit.cs

```diff
@@ -2292,4 +2337,261 @@ interface MKZoomControl
 		MKMapView MapView { get; set; }
 	}
 
+	[Mac (13,0), iOS (16,0), MacCatalyst (16,0), NoWatch, TV (16,0)]
+	[BaseType (typeof (MKMapConfiguration))]
+	[DisableDefaultCtor]
+	interface MKHybridMapConfiguration
+	{
+		[DesignatedInitializer]
+		[Export ("init")]
+		NativeHandle Constructor ();
+
+		[Export ("initWithElevationStyle:")]
+		NativeHandle Constructor (MKMapElevationStyle elevationStyle);
+
+		[NullAllowed, Export ("pointOfInterestFilter", ArgumentSemantic.Copy)]
+		MKPointOfInterestFilter PointOfInterestFilter { get; set; }
+
+		[Export ("showsTraffic")]
+		bool ShowsTraffic { get; set; }
+	}
+
+	[NoMac, iOS (16,0), MacCatalyst (16,0), NoWatch, NoTV]
+	[BaseType (typeof (NSObject))]
+	[DisableDefaultCtor]
+	interface MKIconStyle
+	{
+		[Export ("backgroundColor")]
+		UIColor BackgroundColor { get; }
+
+		[Export ("image")]
+		UIImage Image { get; }
+	}
+
+	[Mac (13,0), iOS (16,0), MacCatalyst (16,0), NoWatch, TV (16,0)]
+	[BaseType (typeof (MKMapConfiguration))]
+	[DisableDefaultCtor]
+	interface MKImageryMapConfiguration
+	{
+		[DesignatedInitializer]
+		[Export ("init")]
+		NativeHandle Constructor ();
+
+		[Export ("initWithElevationStyle:")]
+		NativeHandle Constructor (MKMapElevationStyle elevationStyle);
+	}
+
+	[Mac (13,0), iOS (16,0), MacCatalyst (16,0), NoWatch, NoTV]
+	[BaseType (typeof (NSObject))]
+	[DisableDefaultCtor]
+	interface MKLookAroundScene : NSCopying { }
+
+	[Mac (13,0), iOS (16,0), MacCatalyst (16,0), NoWatch, NoTV]
+	[BaseType (typeof (NSObject))]
+	[DisableDefaultCtor]
+	interface MKLookAroundSceneRequest
+	{
+		[Export ("initWithCoordinate:")]
+		[DesignatedInitializer]
+		NativeHandle Constructor (CLLocationCoordinate2D coordinate);
+
+		[Export ("initWithMapItem:")]
+		[DesignatedInitializer]
+		NativeHandle Constructor (MKMapItem mapItem);
+
+		[Export ("coordinate")]
+		CLLocationCoordinate2D Coordinate { get; }
+
+		[NullAllowed, Export ("mapItem")]
+		MKMapItem MapItem { get; }
+
+		[Export ("isCancelled")]
+		bool IsCancelled { get; }
+
+		[Export ("isLoading")]
+		bool IsLoading { get; }
+
+		[Async]
+		[Export ("getSceneWithCompletionHandler:")]
+		void GetScene (Action<MKLookAroundScene, NSError> completionHandler);
+
+		[Export ("cancel")]
+		void Cancel ();
+	}
+
+	[Mac (13,0), iOS (16,0), MacCatalyst (16,0), NoWatch, NoTV]
+	[BaseType (typeof (NSObject))]
+	interface MKLookAroundSnapshot
```

Comment:
This is used as a return object.

```suggestion
	[DisableDefaultCtor]
	interface MKLookAroundSnapshot
```